frontend/src/metabase/home/containers/HomepageApp.jsx

```diff
@@ -50,29 +45,11 @@ export default class HomepageApp extends Component {
 
   constructor(props) {
     super(props);
-    this.state = {
-      greeting: Greeting.sayHello(props.user && props.user.first_name),
```

Comment:
I'm guessing we don't want to remove this either — right @kdoh?

Reply:
If I'm not mistaken, there is a naming confusion here.
`HomePage.jsx` is actually `/activity` page, while
`Overworld.jsx` is a home page.

There is a `Greeting()` function in Subhead at `frontend/src/metabase/containers/Overworld.jsx:95`.

Reply:
Yeah `greeting` is never accessed in the component state here.

Reply:
@mazameli check this out :)

![image](https://user-images.githubusercontent.com/31325167/102657135-754ea700-4175-11eb-834d-c837b40411a9.png)

Reply:
My bad! Yeah, the Activity page used to be the homepage years ago. :)

Reply:
I'm learning some history here :)